ipinfo/cmd_default.go

```diff
@@ -108,6 +134,12 @@ func cmdDefault() (err error) {
 		return nil
 	}
 
+	if fPage {
```

Comment:
Shouldn't reference 'page' like that - we can just call it `fHelpDetailed` to be more clear.

Function should be similarly renamed